_data/hosting.yml

```diff
@@ -495,3 +495,10 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://wpengine.com/support/2-factor-authentication/
+    
+    - name: bHosted.nl
```

Comment:
Hello @Stefanbs23,
inside our YAML files we use alphabetical order when listing sites. Please put ´bHosted` below `ARP Networks`